### ISSUE TITLE
How could i perform a check on when ever a publisher publishes something 

### ISSUE THREAD

**Issue by ghostcoffins** [martin-ger/uMQTTBroker#4]
hy!
i am using this library a great effort by martin-ger thumbs up for him, now i want to high a gpio pin when ever client publishes the topic

**Comment by RudyFiero** [comment]
The demo example does that already.
`
 Try to connect from a remote client and publish something - **the console will show this as well**.
`
Look at **data_callback**.

**Comment by ghostcoffins** [comment]
yes i can see the data at the console but i want to trigger some gpio high when ever a specific topic is published

**Comment by martin-ger** [comment]
Do a subscription on the specific topic with "MQTT_local_subscribe((unsigned char *)"yourtopic", 0);" and in the "data_callback" check the topic and the data and trigger the gpio.

You might also consider to use my https://github.com/martin-ger/esp_mqtt project, there you can do this with an even simpler script.
